src/robots.njk

```diff
@@ -0,0 +1,12 @@
+---
+permalink: /robots.txt
+eleventyExcludeFromCollections: true
+---
+Sitemap: {{ site[config.defaultLanguage].url }}sitemap.xml
```

Comment:
We should review the variable use here.

Reply:
Should we just hardcode site url for this?